app/main.py

```diff
@@ -1,2 +1,92 @@
+from dataclasses import dataclass
+from typing import Any
+
+
+@dataclass
+class Node:
+    key: Any
+    value: Any
+    hashed_key: int
+
+
 class Dictionary:
-    pass
+    def __init__(self) -> None:
+        self.lenght = 0
+        self.hash_table = [None for i in range(0, 8)]
+
+    def __len__(self) -> int:
+        count = 0
+        for element in self.hash_table:
+            if element is not None:
+                count += 1
+        self.lenght = count
```

Comment:
create `self.lenght` in init and always add one when a new value is written to `self.hash_table`